osbuild.spec

```diff
@@ -34,6 +34,7 @@ Requires:       qemu-img
 Requires:       systemd
 Requires:       tar
 Requires:       util-linux
+Requires:       python3-iniparse
```

Comment:
Doesn't need to be on the host. Stages are run inside a contained environment, and dependencies are only needed to be in the build root.

Reply:
That's a good point, I'll remove it.

Reply:
Done.